libvirt/tests/src/storage/virsh_pool_vol.py

```
@@ -0,0 +1,405 @@
+        pool_define_extra = ""
+        if pool_type == "fs":
+            root_src = process.run(
+                "findmnt -n -o SOURCE /",
+                shell=True, ignore_status=False).stdout_text.strip()
+            root_dev_name = os.path.basename(root_src)
+            root_pkname = process.run(
+                "lsblk -ndo PKNAME %s" % root_src,
+                shell=True, ignore_status=True).stdout_text.strip()
+            root_disk_name = root_pkname or root_dev_name
+            lsblk_cmd = "lsblk -dn -o NAME,TYPE,MOUNTPOINT,PKNAME"
+            lsblk_result = process.run(lsblk_cmd, shell=True, ignore_status=False)
```

Ruff (0.15.18)

[error] Function call with `shell=True` parameter identified, security issue

(S604)